tests/testthat/test-mergingandstackingutilities.R

```diff
@@ -0,0 +1,77 @@
+test_that("variableType", {
+    expect_equal(variableType(variable = 1:10), "Numeric")
+    expect_equal(variableType(variable = LETTERS), "Text")
+    categorical.var <- structure(c(1, 2, 1),
+                                 labels = structure(1:2, .Names = c("A", "B")))
+    expect_equal(variableType(variable = categorical.var), "Categorical")
+    dates <- structure(c(18802, 18803, 18804), class = "Date")
+    expect_equal(variableType(variable = dates), "Date")
+    date.times <- structure(c(1624538040, 1624589160, 1624733100),
+                            class = c("POSIXct", "POSIXt"), tzone = "UTC")
+    expect_equal(variableType(variable = date.times), "Date/Time")
+    duration <- structure(10:12, class = "difftime", units = "mins")
+    expect_equal(variableType(variable = duration), "Duration")
```

Comment:
You should probably change these to user the new constants.  Also I think I saw them in the main file.

Reply:
Done in 2fd7df8